.github/utils/api/splunkbase.py

```diff
@@ -1,11 +1,8 @@
 import logging
-import os
-
+from typing import Optional, Tuple
```

Comment:
Don't need `typing.Tuple` in Python 3.9+
```suggestion
from typing import Optional
```